recipe/meta.yaml

```diff
@@ -1,58 +1,62 @@
 {% set name = "rdkit" %}
-{% set version = "2023.09.1" %}
+{% set version = "2025.03.5" %}
 {% set filename = "Release_%s.tar.gz" % version.replace(".", "_") %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/{{ name }}/{{ name }}/archive/{{ filename }}
-  sha256: e0ff8e330c98b93ac8277a59b2369d9a38027afadb4f03bb34c6924d445f08d5
-  # Fixes outdated checksum for downloaded fonts.
-  # https://github.com/rdkit/rdkit/commit/ba0b6f3bafb391c17d7da9783f09817a4ba0f8f1
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/{{ filename }}
+  sha256: 8bdb3c774ed4ae9e2727b6ce005004191447d630d8e857d36839cd8f1bca55b5
   patches:
-    - fix-install_comic_fonts.patch
+    - 0001_fix_source.patch
 
 build:
   number: 0
   # We skip s390x because it's not supported.
   skip: True # [py<37 or s390x]
   run_exports:
     - {{ pin_subpackage('rdkit', max_pin='x.x.x') }}
+  ignore_run_exports:
+    - numpy
+    - libcxx            # [unix]
+    - ucrt              # [win]
+    - vc14_runtime      # [win]
 
 requirements:
   build:
     - patch  # [not win]
     - m2-patch  # [win]
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
     - python
     - numpy
-    - cmake
+    - cmake >=3.16
+    # Tests are using catch2 which is unavailable on the main channel,
+    # using vendored one instead
+    - git                                    # [linux]
     - jom                                    # [win]
     - make                                   # [unix]
     - pkg-config
   host:
-    - libboost {{ boost }}
-    - py-boost {{ boost }}
-    - eigen
-    - freetype
+    - libboost-devel {{ boost }}
+    - libboost-python-devel {{ boost }}
+    - eigen {{ eigen }}
+    - freetype {{ freetype }}
     - python
+    - pytest
     - numpy {{ numpy }}
-    - pillow
-    - pandas
-    - setuptools
-    - wheel
+    - pillow >=10.4.0
+    - pandas >=2.2.3
+    - setuptools >=58.0.4
   run:
-    - libboost
-    - py-boost
     - python
-    - pillow
-    - pandas
     - {{ pin_compatible('numpy') }}
     - matplotlib-base
     - sqlalchemy
+    - pillow
     - reportlab
```

Comment:
Does it require only numpy and pillow? See https://inspector.pypi.io/project/rdkit/2025.3.5/packages/4c/c3/1968d5684b1874db71137aceef1904fccc822aca2963be0c2a3dee4ee226/rdkit-2025.3.5-cp39-cp39-manylinux_2_28_x86_64.whl/rdkit-2025.3.5.dist-info/METADATA#line.14

Reply:
See also https://github.com/kuelumbus/rdkit-pypi/tree/2025.03.5 and https://github.com/rdkit/rdkit/tree/Release_2025_03_5/Docs#cross-platform-using-pip